packages/client/mutations/CreateTaskMutation.ts

```diff
@@ -121,7 +121,8 @@ const CreateTaskMutation: StandardMutation<TCreateTaskMutation, OptionalHandlers
         createdBy: viewerId,
         updatedAt: now,
         tags: [],
-        content: newTask.content || makeEmptyStr()
+        content: newTask.content || makeEmptyStr(),
+        plaintextContent: newTask.plaintextContent || makeEmptyStr()
```

Comment:
`makeEmptyStr` isn't plaintext, it's a JSON string. probably best to just use `extractTextFromDraftString` on whatever the content is